package.nls.json

```diff
@@ -26,5 +26,7 @@
     "azureApiManagement.showArmProduct": "Edit Product",
     "azureApiManagement.openExtensionWorkspaceFolder": "Open extension workspace folder",
     "azureApiManagement.initializeExtensionWorkspaceFolder": "Initialize extension workspace folder",
-    "azureApiManagement.advancedPolicyAuthoringExperience": "(Experimental Feature) Enables advanced policy authoring experience."
+    "azureApiManagement.advancedPolicyAuthoringExperience": "(Experimental Feature) Enables advanced policy authoring experience.",
+    "azureApiManagement.extractService": "Extract configurations into ARM Templates",
```

Comment:
both the commands will show up with same information in command palette.

Reply:
yep, @miaojiang